packages/docs/docs/cli/external-fhir-servers.md

```diff
@@ -44,14 +44,19 @@ medplum profile set <profileName> \
     --fhir-url-path <fhir-url-path> \
     --token-url <token-url> \
     --client-id <client-id> \
-    --client-secret <client-secret>
+    --client-secret <client-secret> \
+    --assertion <assertion> \
+    --scope <scope> \
+    --access-token <access-token>
 ```
 
 | Accepted Auth Type |
 | ------------------ |
 | basic              |
 | client_credentials |
 | authorization_code |
+| jwt-bearer         |
```

Comment:
why are some dashes (-) and some underscore (_) - are those industry standard things?

Reply:
Ahh good point with normalizing them. Dashes are more accepted with the CLI so I'll make changes for all of them to be that.